diff --git a/build/config-plugin.js b/build/config-plugin.js
index 588870684b4151a62c443542d9077198225fa1b2..d4376806d8abca0321d77b4c8c6b017bf5bb4123 100644
--- a/build/config-plugin.js
+++ b/build/config-plugin.js
@@ -6,7 +6,6 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.withTargetsDir = void 0;
 const glob_1 = require("glob");
 const path_1 = __importDefault(require("path"));
-const chalk_1 = __importDefault(require("chalk"));
 const withPodTargetExtension_1 = require("./withPodTargetExtension");
 const withWidget_1 = __importDefault(require("./withWidget"));
 const withXcparse_1 = require("./withXcparse");
@@ -18,7 +17,7 @@ const withTargetsDir = (config, _props) => {
     const projectRoot = config._internal.projectRoot;
     if (!appleTeamId && !hasWarned) {
         hasWarned = true;
-        console.warn((0, chalk_1.default) `{yellow [bacons/apple-targets]} Expo config is missing required {cyan ios.appleTeamId} property. Find this in Xcode and add to the Expo Config to correct. iOS builds may fail until this is corrected.`);
+        console.warn(`{yellow [bacons/apple-targets]} Expo config is missing required {cyan ios.appleTeamId} property. Find this in Xcode and add to the Expo Config to correct. iOS builds may fail until this is corrected.`);
     }
     const targets = (0, glob_1.globSync)(`${root}/${match}/expo-target.config.@(json|js)`, {
         // const targets = globSync(`./targets/action/expo-target.config.@(json|js)`, {
diff --git a/build/withWidget.js b/build/withWidget.js
index 7f7759060863b60c60f68d3abfcc92d03352d0e2..bc15a7ae1a9e5c888395f146a9d54cc8da92f872 100644
--- a/build/withWidget.js
+++ b/build/withWidget.js
@@ -8,7 +8,6 @@ const plist_1 = __importDefault(require("@expo/plist"));
 const fs_1 = __importDefault(require("fs"));
 const glob_1 = require("glob");
 const path_1 = __importDefault(require("path"));
-const chalk_1 = __importDefault(require("chalk"));
 const withIosColorset_1 = require("./colorset/withIosColorset");
 const withImageAsset_1 = require("./icon/withImageAsset");
 const withIosIcon_1 = require("./icon/withIosIcon");
@@ -53,7 +52,7 @@ function kebabToCamelCase(str) {
 }
 const withWidget = (config, props) => {
     var _a, _b, _c, _d, _e, _f, _g, _h, _j;
-    prebuildLogQueue.add(() => warnOnce((0, chalk_1.default) `\nUsing experimental Config Plugin {bold @bacons/apple-targets} that is subject to breaking changes.`));
+    prebuildLogQueue.add(() => warnOnce(`\nUsing experimental Config Plugin {bold @bacons/apple-targets} that is subject to breaking changes.`));
     // TODO: Magically based on the top-level folders in the `ios-widgets/` folder
     if (props.icon && !/https?:\/\//.test(props.icon)) {
         props.icon = path_1.default.join(props.directory, props.icon);
@@ -95,13 +94,13 @@ const withWidget = (config, props) => {
                     // Then set the target app groups to match the main app.
                     entitlements[APP_GROUP_KEY] = mainAppGroups;
                     prebuildLogQueue.add(() => {
-                        logOnce((0, chalk_1.default) `[${widget}] Syncing app groups with main app. {dim Define entitlements[${JSON.stringify(APP_GROUP_KEY)}] in the {bold expo-target.config} file to override.}`);
+                        logOnce(`[${widget}] Syncing app groups with main app. {dim Define entitlements[${JSON.stringify(APP_GROUP_KEY)}] in the {bold expo-target.config} file to override.}`);
                     });
                 }
                 else {
                     prebuildLogQueue.add(() => {
                         var _a, _b;
-                        return warnOnce((0, chalk_1.default) `{yellow [${widget}]} Apple target may require the App Groups entitlement but none were found in the Expo config.\nExample:\n${JSON.stringify({
+                        return warnOnce(`{yellow [${widget}]} Apple target may require the App Groups entitlement but none were found in the Expo config.\nExample:\n${JSON.stringify({
                             ios: {
                                 entitlements: {
                                     [APP_GROUP_KEY]: [
