diff --git a/ios/Shared/RCTMenuItem.swift b/ios/Shared/RCTMenuItem.swift
index bb6bb2b7ad56135089f267587c974b166760539d..b60c6321ff40393706093d12539d8337609a85c1 100644
--- a/ios/Shared/RCTMenuItem.swift
+++ b/ios/Shared/RCTMenuItem.swift
@@ -6,6 +6,7 @@
 //
 
 import UIKit;
+import SVGKit
 
 @available(iOS 13.0, *)
 class RCTMenuAction {
@@ -32,6 +33,13 @@ class RCTMenuAction {
             if self.image === nil {
                 self.image = UIImage(named: image as String)
             }
+            if self.image === nil {
+                self.image = UIImage();
+                if let url = Bundle.main.url(forResource: image as String, withExtension: "svg") {
+                    let svg = SVGKImage(contentsOf: url)
+                    self.image = svg?.uiImage
+                }
+            }
             if let imageColor = details["imageColor"] {
                 self.image = self.image?.withTintColor(RCTConvert.uiColor(imageColor), renderingMode: .alwaysOriginal)
             }
diff --git a/react-native-menu.podspec b/react-native-menu.podspec
index 5453e381caf9e447bb0fa882d923cf7645ff4e0f..96d310cce6c3f2e84dbdf8c8dea52d7f793f1b62 100644
--- a/react-native-menu.podspec
+++ b/react-native-menu.podspec
@@ -16,6 +16,8 @@ Pod::Spec.new do |s|
 
   s.source_files = "ios/**/*.{h,m,mm,swift}"
 
+  s.dependency 'SVGKit'
+
   if ENV['RCT_NEW_ARCH_ENABLED'] == '1' then # new architecture
     # Use install_modules_dependencies helper to install the dependencies if React Native version >=0.71.0.
     # See https://github.com/facebook/react-native/blob/febf6b7f33fdb4904669f99d795eba4c0f95d7bf/scripts/cocoapods/new_architecture.rb#L79.
