diff --git a/ios/VideoManager.swift b/ios/VideoManager.swift
index 0b82a77c4c2d895ab5c92342935ed1407361222b..d40c715b55e2ab1d313d8c18daa41b540424c91b 100644
--- a/ios/VideoManager.swift
+++ b/ios/VideoManager.swift
@@ -104,7 +104,7 @@ class VideoManager {
 
     if audioSession.categoryOptions != audioSessionCategoryOptions || audioSession.category != .playback || audioSession.mode != .moviePlayback {
       do {
-        try audioSession.setCategory(.playback, mode: .moviePlayback, options: audioSessionCategoryOptions)
+        try audioSession.setCategory(.playback, options: audioSessionCategoryOptions)
       } catch {
         log.warn("[expo-video] Failed to set audio session category. This might cause issues with audio playback and Picture in Picture. \(error.localizedDescription)")
       }
