diff --git a/ios/VideoManager.swift b/ios/VideoManager.swift
index c2ebd04d3830c0fdfb51a0677f6df279d8420613..a98eeb6750d7ece818bd7d44e00722e51090cbef 100644
--- a/ios/VideoManager.swift
+++ b/ios/VideoManager.swift
@@ -149,7 +149,7 @@ class VideoManager {
 
     if audioSession.categoryOptions != audioSessionCategoryOptions || audioSession.category != .playback || audioSession.mode != .moviePlayback {
       do {
-        try audioSession.setCategory(.playback, mode: .moviePlayback, options: audioSessionCategoryOptions)
+        try audioSession.setCategory(.playback, options: audioSessionCategoryOptions)
       } catch {
         log.warn("[expo-video] Failed to set audio session category. This might cause issues with audio playback and Picture in Picture. \(error.localizedDescription)")
       }
