diff --git a/Users/alizahid/Desktop/apps/acorn/node_modules/phosphor-react-native/.bun-tag-d16709439aa53182 b/.bun-tag-d16709439aa53182
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/lib/typescript/lib/index.d.ts b/lib/typescript/lib/index.d.ts
index ed4c3a4a475fcac062c82eca932a566491019ca3..9547ce191910de962458b62a68f6ed7727ff3adb 100644
--- a/lib/typescript/lib/index.d.ts
+++ b/lib/typescript/lib/index.d.ts
@@ -1,5 +1,5 @@
 import React from 'react';
-import type { StyleProp, TextStyle, ViewStyle } from 'react-native';
+import type { StyleProp, ViewStyle } from 'react-native';
 export type IconWeight = 'thin' | 'light' | 'regular' | 'bold' | 'fill' | 'duotone';
 export interface IconProps {
     color?: string;
@@ -7,7 +7,7 @@ export interface IconProps {
     duotoneOpacity?: number;
     mirrored?: boolean;
     size?: string | number;
-    style?: StyleProp<ViewStyle | TextStyle>;
+    style?: StyleProp<ViewStyle>;
     testID?: string;
     title?: string;
     titleId?: string;
diff --git a/src/lib/index.tsx b/src/lib/index.tsx
index f3aa238feeb1e0fca245f6cd84208b9784ba6dd7..da58408cc5bfcaada4ba201068078d9224efcf81 100644
--- a/src/lib/index.tsx
+++ b/src/lib/index.tsx
@@ -1,5 +1,5 @@
 import React, { createContext } from 'react';
-import type { StyleProp, TextStyle, ViewStyle } from 'react-native';
+import type { StyleProp, ViewStyle } from 'react-native';
 
 export type IconWeight =
   | 'thin'
@@ -15,7 +15,7 @@ export interface IconProps {
   duotoneOpacity?: number;
   mirrored?: boolean;
   size?: string | number;
-  style?: StyleProp<ViewStyle | TextStyle>;
+  style?: StyleProp<ViewStyle>;
   testID?: string;
   title?: string; // SVGRProps
   titleId?: string; // SVGRProps
