diff --git a/dist/reactnative/platform/reactnative/scrollcomponent/ScrollComponent.js b/dist/reactnative/platform/reactnative/scrollcomponent/ScrollComponent.js
index 18c5ae2277b3dda55cda1aa65ae7f4110736cc59..e15d430e9b0d536af00ac5618e282eafbc283e68 100644
--- a/dist/reactnative/platform/reactnative/scrollcomponent/ScrollComponent.js
+++ b/dist/reactnative/platform/reactnative/scrollcomponent/ScrollComponent.js
@@ -107,7 +107,7 @@ var ScrollComponent = /** @class */ (function (_super) {
         //     ...props,
         // } = this.props;
         return (React.createElement(Scroller, __assign({ ref: this._getScrollViewRef, removeClippedSubviews: false, scrollEventThrottle: this.props.scrollThrottle }, this.props, { horizontal: this.props.isHorizontal, onScroll: this._onScroll, onLayout: (!this._isSizeChangedCalledOnce || this.props.canChangeSize) ? this._onLayout : this.props.onLayout }),
-            React.createElement(react_native_1.View, { style: { flexDirection: this.props.isHorizontal ? "row" : "column" } },
+            React.createElement(react_native_1.View, { style: { flexDirection: this.props.isHorizontal ? "row" : "column", flexGrow: 1 } },
                 renderContentContainer(contentContainerProps, this.props.children),
                 this.props.renderFooter ? this.props.renderFooter() : null)));
     };
